ipfs-podcasting/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       APP_PORT: $APP_IPFS_PODCASTING_PORT
 
   web:
-    image: ipfspodcasting/podcastnode:v0.5@sha256:aa56c571494323f210677ad4291ffead0fff31a1da1544766557e54b413f43d0
+    image: ipfspodcasting/podcastnode:v0.6
```

Comment:
Could you add the multi-architecture Docker image digest: `sha256:773d7c34d452764be973e8bc495990ec47f8042287493997784198c64e924393`

If you run `docker pull <image>` it will show you the digest at the bottom of the output.

Reply:
Thanks. I confirmed the digest and added to docker-compose.yml